### DISCUSSION TITLE
x-for template has stopped processing content correctly

### DISCUSSION THREAD

**Discussion by NemoKing1210** [alpinejs/alpine#4464] [7. Bugs (archived)]
The following code has stopped working correctly since version 3.14.x

```html
<template x-for="item in navItem?.children" :key="item.label">
     <a class="site-menu-modal-nav__item" :href="item.href" x-text="item.label"></a>
</template>
```

It creates nodes, but the element inside the template stops being processed, which is why the attributes x-bind, x-text and others do not work. He inserts the raw markup.

![image](https://github.com/user-attachments/assets/2a68ed14-dc32-4ca4-90e2-0807a69aafa9)

**Comment by SimoTod** [comment]
Can you post a full example to show the bug? Works okay [here](https://codepen.io/SimoTod/pen/WbeQRvQ)
